gensim/models/ldamodel.py

```diff
@@ -1054,13 +1059,22 @@ def load(cls, fname, *args, **kwargs):
             result.state = super(LdaModel, cls).load(state_fname, *args, **kwargs)
         except Exception as e:
             logging.warning("failed to load state from %s: %s", state_fname, e)
-        id2word_fname = utils.smart_extension(fname, '.id2word')
-        if (os.path.isfile(id2word_fname)):
-            try:
-                result.id2word = utils.unpickle(id2word_fname)
-            except Exception as e:
-                logging.warning("failed to load id2word dictionary from %s: %s", id2word_fname, e)
+
+        random_state_fname = utils.smart_extension(fname, '.random_state')
+        if (os.path.isfile(random_state_fname)):
+            result.random_state = utils.unpickle(random_state_fname)
         else:
-            result.id2word = None
+            logging.warning("random_state not stored on disk so using default value")
```

Comment:
@tmylk I believe I have understood the solution that we want. However, I have a minor doubt about where would the `.random_state` file come from? Is it that the user is responsible for creating it explicitly always and we (i.e. from within the `save` function) need not create it ever?
If this is true, then in case we are loading a pre-0.13.2 model and no `.random_state` file exists on disk, then should we set `result.random_state` using a default value like `get_random_state(None)` with a `logger.warning`?